src/lib/isEmail.js

```diff
@@ -20,7 +20,8 @@ const default_email_options = {
 /* eslint-disable max-len */
 /* eslint-disable no-control-regex */
 const splitNameAddress = /^([^\x00-\x1F\x7F-\x9F\cX]+)</i;
-const emailUserPart = /^[a-z\d!#\$%&'\*\+\-\/=\?\^_`{\|}~]+$/i;
+// Updated regular expression pattern for email user part validation
+const emailUserPart = /^[a-zA-Z0-9!#$%&'*+/=?^_`{|}~\-]+(\.[a-zA-Z0-9!#$%&'*+/=?^_`{|}~\-]+)*$/;
```

Comment:
This is a bigger change than I expected, why is that? What is the source of this new RegEx pattern?

Also; the comment is not needed here